fairlearn/reductions/_grid_search/_grid_generator.py

```diff
@@ -20,11 +20,17 @@
 class _GridGenerator:
     """A generator of a grid of points with a bounded L1 norm."""
 
-    def __init__(self, grid_size, grid_limit, pos_basis, neg_basis, neg_allowed, force_L1_norm):
+    def __init__(self, grid_size, grid_limit, pos_basis, neg_basis, neg_allowed, force_L1_norm,
+                 grid_center=None):
         # grid parameters
         self.dim = len(pos_basis.columns)
         self.neg_allowed = neg_allowed
         self.force_L1_norm = force_L1_norm
+        if grid_center is None:
```

Comment:
Wonderful! Only one remark: can you add a comment about grid_center, explaining what it is, or if you feel very generous you can add a docstring explaining all the input params.
@MiroDudik @riedgar-ms what is force_L1_norm for?

Reply:
lets rename `grid_center` -> `grid_origin` (the grid is not necessarily symmetric). `force_L1_norm=True` ensures that all points of the grid have the L1 norm equal to `grid_limit`; if `force_L1_norm=False` then grid consists of points whose L1 norm is less than equal to `grid_limit`.

Reply:
[this is of course no longer true once we shift the grid to a different origin :-)]

Reply:
Actually, for renaming, let's do `grid_center` -> `grid_offset`. Hopefully the final word on the renaming.